config/aso/kustomization.yaml

```diff
@@ -12,7 +12,7 @@ patches:
       apiVersion: v1
       kind: Namespace
       metadata:
-        name: capz-system
+        name: azureserviceoperator-system
```

Comment:
This looks extremely bad but I promise I was running into issues with this same change while I was first iterating on this lol. It seemed like the `namespace:` above may have been affecting the input to this patch?